config/routes.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
+  # Define root URL for the application
+  root 'pages#index'
+  
+  # Pages routes
+  get 'home' => 'pages#home'
```

Comment:
I'm planning to use the `index` page as the landing page of the application then display `home` once users are signed in.

Reply:
I take it `pages` is for your static pages? In that case, `index` makes sense as your landing page.
However, once the user is logged in, it may not be wise to use a static page for your user dashboard (I'm assuming here that the user gets redirected to a dashboard-like page, but it can also be the user's profile). Instead, use another controller. I suggest `dashboard#show` or `user#show`. Use your pages controller strictly for static pages (e.g., landing page, contact us, privacy policy, etc).

Reply:
I see, this is a TIL for me, thank you po. :smile: